velero-plugin-for-microsoft-azure/object_store.go

```diff
@@ -49,6 +49,8 @@ const (
 	storageAccountConfigKey          = "storageAccount"
 	storageAccountKeyEnvVarConfigKey = "storageAccountKeyEnvVar"
 	blockSizeConfigKey               = "blockSizeInBytes"
+	storageAccountUriConfigKey       = "storageAccountUri"
+	useAADConfigKey                  = "useAAD"
```

Comment:
Please also update the doc https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/blob/main/backupstoragelocation.md for the new items